.changeset/ten-crabs-travel.md

```diff
@@ -0,0 +1,6 @@
+---
+"@aws-amplify/ui-react-geo": patch
+"@aws-amplify/ui-react": patch
```

Comment:
This doesn't impact ui-react i think?

Reply:
+1

Reply:
The changeset automatically generates it

Reply:
idk why let me remove it

Reply:
done